### ISSUE TITLE
Error while applying player data of x! java.lang.NullPointerException: null

### ISSUE THREAD

**Issue by dwi** [Minebench/SyncInv#1]
My bungee setup
- **Lobby** (server-group: lobby, server-name: lobby) - Lobby shouldn't be synced with anything, not sure if I should put SyncInv to lobby or only to servers I want to sync
- **ServerA** (server-group: _survival_, server-name: **_novus_**) - ServerA and ServerB should be synced
- **ServerB** (server-group: _survival_, server-name: **_space_**)

Player had probably a connection isses and got disconnected from ServerA

- Disconnected on 12:43:34 - player left ServerA and was kicked to fallback server Lobby
- Player reconnected back to ServerA on 12:43:40

```
[12:43:34 INFO] dwi lost connection: Disconnected                           
[12:43:34 INFO] [SyncInv] Debug: Sending DATA to group:survival containing 1 objects.                
[12:43:40 INFO] [SyncInv] Debug: Sending GET_LAST_SEEN to group:survival containing 1 objects.
[12:43:40 INFO] [SyncInv] Debug: Received LAST_SEEN from space targeted at novus containing 2 objects
[12:43:40 INFO] dwi[/x:37701] logged in
```

On the exact moment this error occured on ServerB (player was not trying to join ServerB)

```
[12:43:34 INFO] [SyncInv] Debug: Received DATA from novus targeted at group:survival containing 1 objects
[12:43:34 ERROR] [SyncInv] Error while applying player data of dwi!
java.lang.NullPointerException: null
        at net.minecraft.server.v1_12_R1.EntityPlayer.a(EntityPlayer.java:1128) ~[patched_1.12.1.jar:git-Paper-1192]
        at net.minecraft.server.v1_12_R1.EntityLiving.addEffect(EntityLiving.java:694) ~[patched_1.12.1.jar:git-Paper-1192]
        at org.bukkit.craftbukkit.v1_12_R1.entity.CraftLivingEntity.addPotionEffect(CraftLivingEntity.java:270) ~[patched_1.12.1.jar:git-Paper-1192]
        at org.bukkit.craftbukkit.v1_12_R1.entity.CraftLivingEntity.addPotionEffect(CraftLivingEntity.java:260) ~[patched_1.12.1.jar:git-Paper-1192]
        at org.bukkit.craftbukkit.v1_12_R1.entity.CraftLivingEntity.addPotionEffects(CraftLivingEntity.java:277) ~[patched_1.12.1.jar:git-Paper-1192]
        at de.minebench.syncinv.SyncInv.lambda$applyData$0(SyncInv.java:355) ~[?:?]
        at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:58) [patched_1.12.1.jar:git-Paper-1192]
        at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:356) [patched_1.12.1.jar:git-Paper-1192]
        at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:833) [patched_1.12.1.jar:git-Paper-1192]
        at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:424) [patched_1.12.1.jar:git-Paper-1192]
        at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:767) [patched_1.12.1.jar:git-Paper-1192]
        at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:665) [patched_1.12.1.jar:git-Paper-1192]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_144]
```

Running latest PaperSpigot.

**Comment by Phoenix616** [comment]
Such issues seem to occur with changes to the player data that can't be applied to an offline player. (Like the potion data in this case) I'll add that to the "apply only when player is online"-list.

> Lobby shouldn't be synced with anything, not sure if I should put SyncInv to lobby or only to servers I want to sync

You only need to put the plugin on servers that you want to sync.

**Comment by Phoenix616** [comment]
Fixed in commit 2f63365
